easybuild/easyconfigs/l/libzip/libzip-1.10.1-GCCcore-13.2.0.eb

```
@@ -0,0 +1,38 @@
+easyblock = 'CMakeMake'
+
+name = 'libzip'
+version = '1.10.1'
+
+homepage = 'https://libzip.org/'
+description = "libzip is a C library for reading, creating, and modifying zip archives.."
```

Not sure what happened in https://github.com/easybuilders/easybuild-easyconfigs/pull/20283/commits/ccb9c6942a2838782d7ef256443ec2260afb7abf, but let's revert this: 
```suggestion
description = "libzip is a C library for reading, creating, and modifying zip archives."
```